package.json

```diff
@@ -70,20 +71,22 @@
     "uuid": "9.0.0",
     "vue": "^3.1.0",
     "vue-eslint-parser": "9.3.1",
-    "webpack": "5.88.0",
     "vue-loader": "^16.0.0",
+    "webpack": "5.88.0",
     "webpack-cli": "5.1.1",
     "webpack-dev-server": "4.15.1",
     "webpack-merge": "5.9.0"
   },
   "scripts": {
     "clean": "rm -rf ./dist ./node_modules ./package-lock.json ./coverage ./html-test-results ./test-results ./.nyc_output ",
-    "clean-test-lint": "npm run clean; npm install; npm run test; npm run lint; npm run lint:spelling",
+    "clean-test-lint": "npm run clean; npm install; npm run test; npm run lint",
     "start": "npx webpack serve --config ./.webpack/webpack.dev.js",
     "start:coverage": "npx webpack serve --config ./.webpack/webpack.coverage.js",
-    "lint": "eslint example src e2e --ext .js openmct.js --max-warnings=0 && eslint example src --ext .vue",
+    "lint:src": "eslint example src e2e --ext .js openmct.js --max-warnings=0 && eslint example src --ext .vue",
     "lint:spelling": "cspell \"**/*.{js,md,vue}\" --show-context --gitignore",
-    "lint:fix": "eslint example src e2e --ext .js,.vue openmct.js --fix",
+    "lint": "run-p lint:*",
+    "ci": "run-p \"lint:src -- --quiet\" \"lint:spelling -- --fail-fast --no-progress\"",
```

Comment:
Make sure the name is correct. Can change to `ci:lint` for instance @unlikelyzero

Reply:
this is getting pretty complex. The reason we added `--quiet` was so that the official github reporter output wouldn't explode on in-line output on PRs. Maybe that's no longer necessary.

Let's try:
1.  remove the `--quiet` flag with this the previous implementation on github workflows
2. Update a linter with a true positive failure 
3. run `pr:platform` 

and verify that PRs aren't poluted with 12 versions of the same failed linter rule inline on the code